src/registers/model_specific.rs

```diff
@@ -112,6 +123,29 @@ bitflags! {
     }
 }
 
+bitflags! {
+    /// Flags stored in IA32_U_CET (Table-2-2 in Intel SDM Volume 4). The Intel
+    /// SDM-equivalent names are described in parentheses.
+    pub struct CetFlags: u64 {
```

Comment:
Now that this has a neutral name, could you add the corresponding `SCet` MSR? Both MSRs behave in exactly the same way, so you should be able to just copy the implementation from `UCet`.

Reply:
I can do that.